Library/Formula/libcppa.rb

```diff
@@ -2,8 +2,8 @@ class Libcppa < Formula
   # TODO: since libcppa has been renamed to CAF, this formula should eventually
   # be renamed to 'caf.rb'.
   homepage "http://actor-framework.org/"
-  url "https://github.com/actor-framework/actor-framework/archive/0.12.1.tar.gz"
-  sha1 "a8267a2f5ab4f9124ab0b10279a6600c2867243b"
+  url "https://github.com/actor-framework/actor-framework/archive/0.12.2.tar.gz"
```

Comment:
> this PR should be re-built by the bot, does the “test this please” trick works if we’re not a maintainer?

It doesn't. I spent a decent amount of time early on asking the bot to re-test stuff for me and getting ignored by it :smile_cat: